### DISCUSSION TITLE
Docker Image?

### DISCUSSION THREAD

**Discussion by ThreeCO** [memoetapp/memoet#43] [Q&A] [answered]
Hello dear developers,
is there any possibility to install memoet via docker? Do you know of successful installations anywhere?

I would love to self-host memoet on my Synology NAS.

**Comment by manhtai** [comment, marked as answer]
Hello, let me create an issue for this and start an experiment!

Link: #44

**Reply by manhtai** [reply]
I've added Dockerfile now. Enjoy!

**Reply by ThreeCO** [reply]
Wow. That was quick. Everything works like a charm. 
THANK YOU. 

Maybe just one small idea for your newly added notes on Docker environment variables:
I could not find information why and how to use SECRET_KEY_BASE. In the error logs I could see I need to provide a long enough string. Maybe you just add this in README.md?

**Reply by manhtai** [reply]
It's for encrypting cookies and the like. You can generate it with `openssl rand -hex 48`, I will update README soon.
